compact/far_backward/beamline_electron.xml

```diff
@@ -0,0 +1,102 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Dhevan Gangadharan, Simon Gardner -->
+
+<lccdd>
+
+  <comment>
+     Outgoing electron beam pipes
+     Implemented to allow overlap checks with the far backwards systems
+  </comment>
+
+  <detectors>
+
+    <detector id="Pipe_to_Q1eR_ID" name="Pipe_to_Q1eR" type="BackwardsBeamPipe">
+      <Pipe wall_thickness="2*mm" outerD1="Q1eR_InnerRadius*2" outerD2="Center_Beampipe_Rad*2"
+        end1z="Center_Beampipe_End" end2z="Q1eR_CenterPosition+Q1eR_Length/2"/>
+    </detector>
```

Comment:
Did you mean to remove the `id` fields for these magnets too, and/or place them in a single detector?

Reply:
I didn't mean to remove/merge these IDs but I can if you want.  These were introduced by someone else.

Reply:
It's all collaboration software, no ownership implied. I think that change makes sense to include here, in particular as you are adapting this file. Sorry to tack this on.

Reply:
Ok, I chained together the other far-bwd beampipes and removed the IDs.
I didn't chain together the other far-bwd magnets because they are designed slightly differently.